cadence-style-guide.md

```diff
@@ -0,0 +1,65 @@
+# Cadence <- Style Guide
+## Table of contents
+1. [Identation](#identation)
+1. [Line length](#line-length)
+1. [Comments](#comments)
+1. [Whitespace between code lines](#whitespace-between-code-lines)
+1. [Declaring variables and constants](#whitespace-between-code-lines)
+1. [Bracket spaces](#bracket-spaces)
+1. [Panic messages](#panic-messages)
+1. [Naming and capitalization](#naming-and-capitalization)
+## Identation
+Use 4 spaces per indentation level.
+Prefer spaces over tabs as indentation method, mixing them should be avoided.
+## Line length
+A line lenght of 80 characters is recomended for Cadence. This could be particurally challenging when facing capability-related lines, for instance:
```

Comment:
```suggestion
A line length of 80 characters is recommended for Cadence. This could be particularly challenging when writing capability-related lines, for instance:
```